server/pbenchinacan/load_keycloak.sh

```diff
@@ -104,3 +145,12 @@ curl -i -X POST "${KEYCLOAK_HOST_PORT}/admin/realms/${REALM}/users/${USER_ID}/ro
   -H "Authorization: Bearer ${ADMIN_TOKEN}" \
   -H "Content-Type: application/json" \
   -d '[{"id":"'${ROLE_ID}'","name":"ADMIN"}]'
+
+sleep 5
```

Comment:
Could this be more "programmatic" than blindly sleeping 5 seconds? Someday we're going to get caught by a slow executor.

Reply:
Yeah I can try to query the user to see if the client role is properly set to the user before pausing for the commit, instead of sleeping 5 seconds.

Reply:
By the way, I don't expect this script to run with every PR, rather only when we make a change to the Keycloak configuration should we run this script to update the image in our internal image registry. Because I think very rarely would come a time when we would want to make a configurational change to our committed Keycloak image.

Reply:
That's definitely something to work towards. Right now we don't really have logic to segment our CI, but we'll probably want to consider looking for updates in agent vs server vs dashboard vs container configuration and only building -- and testing -- what's changed. It doesn't really matter, here, though, because any time we try to pull this off we want to be sure we're not giving up early.

Reply:
Sigh! I did try to check if the user has the appropriate role by querying the user id and the CLI command successfully returned the correct role id, however, if I pause and commit the image immediately without waiting for at least 2 seconds it doesn't retain that configuration for some reason. So I will keep this delay of 2 seconds, for now, to make sure all the configurations are properly in place when we commit the image.

Reply:
So we can verify that the information is available, but "something" isn't stable for saving the state for a few additional seconds? Ugh, that's terrible. This step bugs me, because sooner or later we're going to have a problem... and we won't even know why. There's nothing we can do to characterize and contain this risk?

Reply:
Removed `podman commit` all it does now is pull the base image and run the configuration